CHANGELOG.md

```diff
@@ -0,0 +1,427 @@
+# Changelog
+
+#### 2.35.1
+
+Release date: (July 12, 2017)
+
+-   Prevent NullPointerException when Parameterized Trigger starts
+    multiple builds and rejects some of them due to the missing
+    Job/Build permission
+    ([JENKINS-45471](https://issues.jenkins-ci.org/browse/JENKINS-45471))
+
+#### 2.35
+
+Release date: (July 10, 2017)
+
+-   [Fix security
```

Comment:
Strange linebreak

Reply:
Pandoc did it. Did not want to spend too much time on polishing the changelog

Reply:
it renders fine 👍